book/src/install.md

```diff
@@ -218,10 +218,11 @@ RUSTFLAGS="-C target-feature=-crt-static"
 
 The **runtime** directory is one below the Helix source, so either set a
 `HELIX_RUNTIME` environment variable to point to that directory and add it to
-your `~/.bashrc` or equivalent:
+your `~/.bashrc` or equivalent (you need to expand the full path, using ```~```
+in this configuration is not parsed by helix):
```

Comment:
As far as I know this is usually handled by your shell:

```bash
$ FOO=~/src/helix
$ echo $FOO
/home/michael/src/helix
```

Maybe when you were doing this you were quoting the value? That will stop the shell from expanding the path:

```bash
$ FOO="~/src/helix"
$ echo $FOO
~/src/helix
```

Reply:
I think it would be ok to pass the value of `HELIX_RUNTIME` through `expand_tilde`: as you say it looks pretty awkward when `--health` says that `~/src/helix/runtime` doesn't exist

Reply:
Ah yeah, we want to move the path utilities in helix-core into a lower "stdx" crate that has extensions to the standard library. Let's merge this with just the change to add `export` and I'll add a follow-up issue to expand the tilde once we create the stdx crate (which I'm working on a PR for now)